phys2bids/phys2bids.py

```diff
@@ -132,12 +132,14 @@ def phys2bids(filename, info=False, indir='.', outdir='.', heur_file=None,
     # #!# This can probably be done while parsing?
     outdir = utils.check_input_dir(outdir)
     utils.path_exists_or_make_it(outdir)
-
+    # generate extra path
+    extra_dir = outdir + "/extra"
```

Comment:
```suggestion
    extra_dir = outdir + "/extras"
```

Reply:
I'm not sure if this should be called `extras`, `others`, `phys2bids_others`...

What do you think @smoia ?

Reply:
I don't mind changing the name, but let's decide and then i will do the hole change, because it is also in the integration tests

Reply:
`phys2bids_logs`? `bids_ignore`? `extras`?

Reply:
It's not just the logs in the folder so I'd avoid the word "logs". `bids_ignore` sounds good to me.

Reply:
perfect